data/json/npcs/BGSS_talk_professions.json

```diff
@@ -0,0 +1,410 @@
+[{
+    "id": "TALK_LEADER",
+    "type": "talk_topic",
+    "responses": [
+	{ "text": "<BGSS_intro_question>", 
```

Comment:
Response lines for all the other STORY1 topics needed.

It might be worthwhile for me to build some smarts into npctalk.cpp, so that these copy-paste response lines are autogenerated.  Something like you add a tag to the story1 topics with the required trait, and then TALK_LEADER reads through all off them and adds the appropriate one.

Reply:
I'd appreciate that... 
If I set it up like this:
`    "id": "BGSS_CONFUSED_1_STORY1",
    "type": "talk_topic",
    "trigger_on_trait": "BGSS_confused_1",
    "talk": "<BGSS_intro_question>",
    "dynamic_line":`

would that provide the infrastructure you'd need to set it up?